Common/Camera.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using OpenTK;
+
+namespace LearnOpenGL_TK.Common
+{
+    //The camera class can be static as most games will only need one camera
```

Comment:
I don't really agree with this, we should talk this over on Discord next time you're free.